docs/system/charge_controller.md

```diff
@@ -1,9 +1,94 @@
+
+### Maximum Power Point Tracker
+
+<fig-caption src="system/mppt-charge-controller.svg" caption="MPPT charge controller" num="1" />
+
+MPPT (Maximum Power Point Tracking) controllers contain a DC-DC converter which matches varying voltage at the solar panel input with the output at the battery side.
+
+Because of the typically high conversion efficiencies (>95%), power at input and output remains the same whereas the voltage and current varies correspondingly. In order to get the best out of solar panels, MPPT charge controller finds and operates at optimal current-voltage point. In contrast, PWM controllers operate at the voltage equal to that of battery voltage which is not an optimal current-voltage point resulting in wastage of power. There are many ways to implement the tracking, two will be described here.
+
+#### Fractional Open Circuit Voltage
+
+This methods uses a simple linear relationship between the open circuit and maximum powerpoint voltage:
+
+$$V_{MPP} = k \cdot V_{oc}$$
+
+The factor $k$ can be determined for a given solar panel than used to track the maximum. It is dependant on the panels' temperature though and has to be corrected accordingly. This method is simple to implement though the factor and its temperature dependency has to be adapted to every installation so that the controller itself is not that versatile.
+
+#### Perturb and Observe
+
+As described in the [DC/DC-Converter chapter](../development/dcdc_converter), a longer duy cycle corresponds to higher input voltage at the converter which is also the voltage across the panel. With this method, the converter will start to draw small amounts of power with a small duty cycle and closely measure the power output. It will increase the duty cycle and measure again. If the output is increased, it will further increase the duty cycle and so on until the power decreases and the duty cycle is decreased as well. With a high frequeny, this method will find the MPP but will also always move away again and then jump back. Though it will not always be precisely on the MPP, it will stay very close to it and does not need any temperature adjustmenents.
+
+### PWM Series Regulator
+
+<fig-caption src="system/pwm-charge-controller.svg" caption="PWM charge controller" num="2" />
+
+PWM (Pulse Width Modulation) charge controller is in pratice a switch connected between solar panel and a battery. Instead of providing a steady output, it sends out a series of charging pulses to the battery. The controller constantly checks the state of the battery to determine the frequency and duty cycle of the pulse. In a fully discharged battery, the pulses would be continuous while at the end of the battery charging cycles, pulses get very short to reduce the average current. The controller checks the state of charge on the battery between pulses and adjusts itself each time.
+
+### Other types
+
+#### On/off regulators
+
+#### Shunt regulators
```

Comment:
Please fill with content or remove.

See here for further information on these types: https://iea-pvps.org/wp-content/uploads/2020/01/rep3_05.pdf

Reply:
What do you prefer?

Reply:
I'm fine with removing it. Those types are not relevant anymore today.